packages/xo-web/src/xo-app/sr/tab-general.js

```diff
@@ -238,6 +240,22 @@ export default class TabGeneral extends Component {
     }
   )
 
+  _generateLink = createSelector(this._getDiskGroups, diskGroups => childIds =>
```

Comment:
Still not a semantically correct name. This function doesn't generate a URL. It generates the function that will generate the URL.